src/sensors.cpp

```diff
@@ -391,57 +411,57 @@ void Sensors::calibrate_accel(void)
 
   if (accel_calibration_count_ > 1000)
   {
-    // The temperature bias is calculated using a least-squares regression.
-    // This is computationally intensive, so it is done by the onboard computer in
-    // fcu_io and shipped over to the flight controller.
-    turbomath::Vector accel_temp_bias =
-    {
-      rf_.params_.get_param_float(PARAM_ACC_X_TEMP_COMP),
-      rf_.params_.get_param_float(PARAM_ACC_Y_TEMP_COMP),
-      rf_.params_.get_param_float(PARAM_ACC_Z_TEMP_COMP)
-    };
-
-    // Figure out the proper accel bias.
-    // We have to consider the contribution of temperature during the calibration,
-    // Which is why this line is so confusing. What we are doing, is first removing
-    // the contribution of temperature to the measurements during the calibration,
-    // Then we are dividing by the number of measurements.
-    turbomath::Vector accel_bias = (acc_sum_ - (accel_temp_bias * acc_temp_sum_)) /
-                                    static_cast<float>(accel_calibration_count_);
-
-    // Sanity Check -
-    // If the accelerometer is upside down or being spun around during the calibration,
-    // then don't do anything
-    if ((max_- min_).norm() > 1.0)
-    {
-      rf_.comm_manager_.log(CommLink::LogSeverity::LOG_ERROR, "Too much movement for IMU cal");
-      calibrating_acc_flag_ = false;
-    }
-    else
-    {
-      // reset the estimated state
-      rf_.estimator_.reset_state();
-      calibrating_acc_flag_ = false;
-
-      if (accel_bias.norm() < 3.0)
+      // The temperature bias is calculated using a least-squares regression.
```

Comment:
Did anything actually change here, or is it just indentation changes? Some editor auto-indent everything? (Let's revert to 2-space indent to make changes easier to see / follow style guide)

Reply:
yeah, IDE nonsense.